src/validation.py

```
@@ -0,0 +1,53 @@
+        if not subj_uri.startswith("http"): subj_uri = f"http://dbpedia.org/resource/{subj_uri.replace(' ', '_')}"
+        if not obj_uri.startswith("http"): obj_uri = f"http://dbpedia.org/resource/{obj_uri.replace(' ', '_')}"
+
+        query = f"""
+        SELECT ?p WHERE {{
+            <{subj_uri}> ?p <{obj_uri}> .
+        }}
+        """
+        params = {"query": query, "format": "json"}
+        
+        try:
+            response = requests.get(self.endpoint, params=params, verify=False, timeout=5)
```

Ruff (0.15.0)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)